templates/pages/login.jsx

```diff
@@ -39,23 +38,28 @@ var Login = React.createClass({
 
     return (
       <div>
-        <Header className="desktopHeader" redirectQuery={queryObj} />
-        <Header className="mobileHeader" redirectLabel="Signup" redirectPage="signup" redirectQuery={queryObj} mobile />
+        <Header origin="Login" className="desktopHeader" redirectQuery={queryObj} />
+        <Header origin="Login" className="mobileHeader" redirectLabel="Signup" redirectPage="signup" redirectQuery={queryObj} mobile />
 
         <div className="loginPage innerForm centerDiv">
-          <Form ref="userform" fields={fieldValues} validators={fieldValidators} />
+          <Form ref="userform" fields={fieldValues} validators={fieldValidators} origin="Login" />
           <button onClick={this.processFormData} className="btn btn-awsm">{buttonText}</button>
-          <Link to="reset-password" query={queryObj} className="need-help">Forgot your password?</Link>
+          <Link onClick={this.handleGA.bind(this, 'Forgot your password')} to="reset-password" query={queryObj} className="need-help">Forgot your password?</Link>
         </div>
       </div>
     );
   },
   processFormData: function() {
     var form = this.refs.userform;
+    ga.event({category: 'Login', action: 'Start login'})
     form.processFormData(this.handleFormData);
   },
+  handleGA: function(name) {
+    ga.event({category: 'Login', action: 'Clicked on ' + name + ' link.'});
+  },
   handleFormData: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Login', action: 'Error during form validation'})
```

Comment:
Actually, Ignore this comment. I see we are doing this in `templates/components/form/form.jsx`